diff --git a/app/src/main/java/eu/kanade/presentation/more/settings/screen/SettingsDataScreen.kt b/app/src/main/java/eu/kanade/presentation/more/settings/screen/SettingsDataScreen.kt
index d5f4342..127bc75 100644
--- a/app/src/main/java/eu/kanade/presentation/more/settings/screen/SettingsDataScreen.kt
+++ b/app/src/main/java/eu/kanade/presentation/more/settings/screen/SettingsDataScreen.kt
@@ -108,7 +108,7 @@ object SettingsDataScreen : SearchableSettings {
         val backupPreferences = Injekt.get<BackupPreferences>()
         val storagePreferences = Injekt.get<StoragePreferences>()
         val syncPreferences = Injekt.get<SyncPreferences>()
-        val syncService by syncPreferences.syncService().collectAsState()
+        val syncService by syncPreferences.syncService.collectAsState()
 
         return persistentListOf(
             getStorageLocationPref(storagePreferences = storagePreferences),
@@ -288,16 +288,15 @@ object SettingsDataScreen : SearchableSettings {
     @Composable
     private fun getSyncGroup(syncPreferences: SyncPreferences, syncService: Int): Preference.PreferenceGroup {
         val context = LocalContext.current
-        val scope = rememberCoroutineScope()
         val navigator = LocalNavigator.currentOrThrow
         val syncServiceType = SyncManager.SyncService.fromInt(syncService)
-        val lastSync by syncPreferences.lastSyncTimestamp().collectAsState()
+        val lastSync by syncPreferences.lastSyncTimestamp.collectAsState()
 
         return Preference.PreferenceGroup(
             title = stringResource(MR.strings.pref_sync_service_category),
             preferenceItems = persistentListOf(
                 Preference.PreferenceItem.ListPreference(
-                    preference = syncPreferences.syncService(),
+                    preference = syncPreferences.syncService,
                     title = stringResource(MR.strings.pref_sync_service),
                     entries = persistentMapOf(
                         SyncManager.SyncService.NONE.value to stringResource(MR.strings.off),
@@ -308,12 +307,12 @@ object SettingsDataScreen : SearchableSettings {
             ) + if (syncServiceType == SyncManager.SyncService.SYNCYOMI) {
                 persistentListOf(
                     Preference.PreferenceItem.EditTextPreference(
-                        preference = syncPreferences.syncHost(),
+                        preference = syncPreferences.clientHost,
                         title = stringResource(MR.strings.pref_sync_host),
                         subtitle = stringResource(MR.strings.pref_sync_host_summ),
                     ),
                     Preference.PreferenceItem.EditTextPreference(
-                        preference = syncPreferences.syncAPIKey(),
+                        preference = syncPreferences.clientAPIKey,
                         title = stringResource(MR.strings.pref_sync_api_key),
                         subtitle = stringResource(MR.strings.pref_sync_api_key_summ),
                     ),
